Forge/src/main/java/me/ferdz/placeableitems/network/PlaceableItemsPacketHandler.java

```diff
@@ -0,0 +1,39 @@
+package me.ferdz.placeableitems.network;
+
+import me.ferdz.placeableitems.PlaceableItems;
+import me.ferdz.placeableitems.client.network.ClientPacketHandler;
+import me.ferdz.placeableitems.network.handler.AnonymousPacketHandler;
+import me.ferdz.placeableitems.network.handler.ServerPacketHandler;
+
+import net.minecraft.util.ResourceLocation;
+
+import net.minecraftforge.fml.DistExecutor;
+import net.minecraftforge.fml.network.NetworkRegistry;
+import net.minecraftforge.fml.network.simple.SimpleChannel;
+
+/**
+ * A packet handler for the PlaceableItems mod.
+ *
+ * @author Parker Hawke - Choco
+ */
+public final class PlaceableItemsPacketHandler {
```

Comment:
I'm confused by the whole network handling. I don't believe it's necessary

Reply:
See the above comment on the ClientPacketHandler as it explains its purpose.